klippy/extras/htu21d.py

```diff
@@ -0,0 +1,235 @@
+# Support for HTU21D(F) i2c based temperature sensors
+# Datasheets: https://cdn-shop.adafruit.com/datasheets/1899_HTU21D.pdf
+#
+# Compatible Sensors
+#       HTU21D - Tested
+#       Si7013 - Untested
+#       Si7020 - Untested
+#       Si7021 - Untested
+#       SHT21  - Untested
+#
+# NOTE: Implementation need the support od 0 length send before read on MCU
+#
+# Copyright (C) 2020  Lucio Tarantino <lucio.tarantino@gmail.com>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
```

Comment:
Please change this so the location of the copyright is similar to other files:
```
# One line description
#
# Copyright ...
# 
# This file may be distributed under the terms of the GNU GPLv3 license.

# Any other comments as needed...
```

Reply:
Done in commit 2cd7404